README

```diff
@@ -862,10 +862,6 @@ performance.threading.internal_handler.timer_msec
 Control XLIO internal thread wakeup timer resolution (in milliseconds). Maps to **XLIO_TIMER_RESOLUTION_MSEC** environment variable.
 Default value is 10
 
-performance.threading.internal_handler.wakeup_per_packet
```

Comment:
make sure to remove this line from the README as well:
"XLIO DETAILS: Internal Thread Arm CQ         Disabled                   [performance.threading.internal_handler.wakeup_per_packet]"

alas, we don't auto-generate currently the header example - maybe we should!